account_invoice_open_sale/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,20 @@
+* Stefan Rijnhart <stefan@opener.amsterdam> (https://opener.amsterdam)
+* Martin Pishpecki <pishpecki@gmail.com> (https://www.vanmoof.com)
+
```

Comment:
You can remove the stuff below. I now remember that the reason to split it up was so that the boilerplate below can be updated at will without all the modules having to be modified manually. Instead, change the boilerplate in the template and regenerate all the readmes automatically.